api/src/apollo/Org/mutations.ts

```diff
@@ -0,0 +1,101 @@
+import { idArg, inputObjectType, mutationField, nonNull } from 'nexus';
+
+export const OrgCreateInputType = inputObjectType({
+  name: 'OrgCreateInputType',
+  definition(t) {
+    t.nonNull.string('name');
+    t.nonNull.string('description');
+    t.nonNull.status('logo');
```

Comment:
Is it a typo? should be string ig?

Reply:
yes it is... status is a different type.. will update it.